### PR TITLE
[Ubuntu] Add clang-format to the readme

### PR DESCRIPTION
**Pull request by miketimofeev**
# Description
We install clang-format along with the clang but don't mention it in the readme.

#### Related issue:
https://github.com/actions/virtual-environments/issues/3053

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 1]
```diff
@@ -20,19 +20,36 @@ function Get-FortranVersions {
     return "GNU Fortran " + ($fortranVersions -Join ", ")
 }
 
-function Get-ClangVersions {
-    $clangVersions = @()
+function Get-ClangTool {
```
Comment (AlenaSviridenko) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:23]: I think `Get-ClangToolVersion` would be more clear name

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 2]
```diff
@@ -20,19 +20,36 @@ function Get-FortranVersions {
...
+    param (
+        [Parameter(Mandatory = $true)]
+        [string] $ToolName,
+        [string] $VersionPattern = "\d+\.\d+\.\d+)-"
+    )
+
     $result = Get-CommandResult "apt list --installed" -Multiline
-    $clangVersions = $result.Output | Where-Object { $_ -match "^clang-\d+"} | ForEach-Object {
+    $clangVersions = $result.Output | Where-Object { $_ -match "^${ToolName}-\d+"} | ForEach-Object {
```
Comment (AlenaSviridenko) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:31]: `$toolVersions` maybe?

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 (hunk 1 above)
Comment (miketimofeev) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:23]: Agree, thanks!

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 (hunk 2 above)
Comment (miketimofeev) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:31]: Maybe, maybe 🤔

**Review by AlenaSviridenko — APPROVED** [review]

**Pull request merged**